e2e/nodebootstrapping_template.json

```diff
@@ -133,6 +133,7 @@
     "EnableGPUDevicePluginIfNeeded": false,
     "EnableKubeletConfigFile": false,
     "EnableNvidia": false,
+    "DisableUnattendedUpgrades": true,
```

Comment:
will be relying on this in conjunction with https://github.com/Azure/AgentBaker/pull/2353/files to disable UU in e2e